README.md

```diff
@@ -1,99 +1,88 @@
-# Carousel
+# VTEX Carousel
+
+## Description
+The VTEX Carousel is an app that shows a collection of banners and it is used by the Dreamstore product.
```

Comment:
See changes in minicart master. You need to update from the changes that @brunojdo did yesterday.